.github/workflows/ci.yaml

```diff
@@ -16,19 +15,20 @@ jobs:
         - '2.6'
         - '2.7'
         - '3.0'
+        - '3.1'
     env:
-      CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+      CC_TEST_REPORTER_ID: "${{ secrets.CC_TEST_REPORTER_ID }}"
     steps:
     - uses: actions/checkout@v4
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: ${{ matrix.ruby-version }}
+        ruby-version: "${{ matrix.ruby-version }}"
         bundler-cache: true
       timeout-minutes: 30
     - name: Run tests
       run: bundle exec rake
     - name: Report code coverage
-      if: ${{ github.ref == 'refs/heads/master' && matrix.ruby-version == '2.7' }}
+      if: "${{ github.ref == 'refs/heads/master' && matrix.ruby-version == '2.7' }}"
```

Comment:
This should probably bump to 3.0 as the new default, but since 2.7 is still in the list this is probably fine.